src/modifiers.rs

```diff
@@ -1,30 +1,49 @@
-//! Defines a series of convenience modifiers for editing Responses
+//! This module defines a series of convenience modifiers for changing
+//! Responses.
 //!
-//! Modifiers can be used to edit Responses through the owning
-//! method `set` or the mutating `set_mut`, both of which are
-//! defined in the `Set` trait.
+//! Modifiers can be used to edit `Response`s through the owning method `set`
+//! or the mutating `set_mut`, both of which are defined in the `Set` trait.
 //!
-//! Instead of having a combinatorial explosion of Response methods
-//! and constructors, this provides a series of modifiers that
-//! can be used through the `Set` trait.
+//! Instead of having a combinatorial explosion of Response methods and
+//! constructors, this provides a series of modifiers that can be used through
+//! the `Set` trait. This is more extensible as it allows you to combine
+//! arbitrary modifiers without having a massive number of Response
+//! constructors.
```

Comment:
@SkylerLipthay I moved the paragraph up here, but still think that the wording is a bit confusing. Maybe you or @steveklabnik have a better proposal on wording?